src/Resources/app/storefront/src/checkout/swag-paypal.acdc-fields.js

```diff
@@ -188,10 +188,7 @@ export default class SwagPaypalAcdcFields extends SwagPaypalAbstractStandalone {
                 return;
             }
 
-            const buttonLoadingIndicator = new ButtonLoadingIndicator(
-                DomAccess.querySelector(this.confirmOrderForm, this.options.confirmOrderButtonSelector),
-            );
-            buttonLoadingIndicator.remove();
+            this.confirmOrderForm.dispatchEvent(new CustomEvent('removeLoader'));
```

Comment:
If we have no fallback, we should backport the platform PR to 6.7.0.x, otherwise we're not fully backwards compatible